fluentd/config.sls

```diff
@@ -24,6 +24,17 @@ create_directory_for_{{ name }}_logs:
      - group
 {% endfor %}
 
+{% for cert, details in salt.pillar.get('fluentd:cert', {}).items() %}
+create_{{ cert }}_file:
+  file.managed:
+    - name: {{ details['path'] }}
```

Comment:
I would use `.` attribute access rather than the bracket lookups. Works either way, but we typically use the former style in Jinja throughout our formulas.